src/main/java/org/owasp/esapi/reference/validation/HTMLValidationRule.java

```diff
@@ -97,8 +97,33 @@ public String sanitize( String context, String input ) {
 		return safe;
 	}
 
+    /**
+     * This admittedly is an UGLY hack to ensure that issue 509 and it's corresponding
+     * fix in PR #510 does not break existing developer's existing code. Full
+     * details are described in GitHub issue 521.
+     */
+    private boolean ignoreIssue509Fix() {
+        boolean ignore = true;
```

Comment:
I don't feel strongly, but I feel like this should default to the "correct" behavior and default to false?  Otherwise we risk new users becoming accustomed to the incorrect behavior?

I suggest making this property "Opt in to be unsafe"